### PR TITLE
Nadir increase to max players allowed (60 > 70)

### PR DESCRIPTION
**Pull request by Kubius**
[INPUT][BALANCE]
## About the PR 

Increases the MaxPlayersAllowed setting for Nadir from 60 to 80, placing it in a similar bracket to other midpop maps like Cog1 and Kondaru.

## Why's this needed? 

Allows Nadir to show up in more circumstances, particularly on the Classic server which tends to run at a higher populace.

While the map was originally added at a cap of 60, I think 80 is pretty reasonable, with the following thoughts in mind:

- During official testing, before any of the improvements and size increases to the map (not even door shields in the first round!), population reached about 80 with things not flying off the handle anywhere near as much as you'd expect.
- Since launch, several significant changes have occurred that make the map more effective at handling larger populace, aside from the door shields:
- A nuclear engine and mining department were recently added, including two new Miner job slots, improving the amount of Stuff engineers can do and increasing the overall crew complement.
- The Hydroponics department was expanded substantially, making it considerably more comfortable for the five-Botanist complement Nadir permits.
- East and west secondary hallways (and maintenance counterparts to the north) were extended one tile wider, alleviating one of the most problematic pinch points that came up during initial testing.
- Four new "auxiliary compartments" (outer 3x5 random rooms) and three new internal random rooms have been added, making for several new spots to take over for projects or hide away for Crime Activities.

## Changelog 

```changelog
(u)Kubius
(+)Nadir's maximum rated occupancy has increased to 70.
```

### CODE REVIEW

**Comment by ZeWaka** [comment]
@Tarmunora

**Review by ZeWaka — APPROVED** [review]

**Comment by ZeWaka** [comment]
eee

File: code/map.dm [hunk 1]
```diff
@@ -31,7 +31,7 @@ var/global/list/mapNames = list(
 	"1 pamgoC" =			list("id" = "PAMGOC",		"settings" = "pamgoc",			"playerPickable" = FALSE),
 	"Kondaru" =				list("id" = "KONDARU",		"settings" = "kondaru",			"playerPickable" = TRUE,	"MaxPlayersAllowed" = 80),
 	"Ozymandias" =			list("id" = "OZYMANDIAS",	"settings" = "ozymandias",		"playerPickable" = FALSE,	"MinPlayersAllowed" = 40),
-	"Nadir" =				list("id" = "NADIR",		"settings" = "nadir",			"playerPickable" = TRUE,	"MaxPlayersAllowed" = 60),
+	"Nadir" =				list("id" = "NADIR",		"settings" = "nadir",			"playerPickable" = TRUE,	"MaxPlayersAllowed" = 80),
```
Comment (ZeWaka) [inline code/map.dm:34]:
```suggestion
	"Nadir" =				list("id" = "NADIR",		"settings" = "nadir",			"playerPickable" = TRUE,	"MaxPlayersAllowed" = 70),
```
let's try 70

**Review by ZeWaka — APPROVED** [review]

**Comment by Kubius** [comment]
Works for me.

**Pull request merged**